sympy/core/mul.py

```diff
@@ -1153,23 +1153,28 @@ def _eval_is_negative(self):
         return (-self).is_positive
 
     def _eval_is_odd(self):
+        from sympy import sqrt
         is_integer = self.is_integer
 
         if is_integer:
             r, acc = True, 1
-            for t in self.args:
-                if not t.is_integer:
+            for t1, t2 in enumerate(self.args):
+                if not t2.is_integer:
+                    if t2.is_Rational:
+                        other = self.args[t1 ^ 1]  # get 0 if 1 or 1 if 0
```

Comment:
Actually, because of [this](https://github.com/sympy/sympy/pull/9258/files#r30987928), `t1` will always be 0.